src/Config.php

```diff
@@ -78,4 +78,25 @@ public function shouldSkipRegionValidation(): bool
     {
         return $this->getValue(['parameters', 'skipRegionValidation']);
     }
+
+    public function isSourceByodb(): bool
+    {
+        return $this->getValue(['parameters', 'preserveTimestamp']);
```

Comment:
Tady ti všude zůstal `preserveTimestamp` param. 🙂

Reply:
tvl 🤦 .  napisu nejake test

Reply:
To by bylo ideální. 🙂

Reply:
@romantmb ma to byt aspon trosku lepsi